app/javascript/packs/application.js

```diff
@@ -1,5 +1,11 @@
 import loadPolyfills from '../mastodon/load_polyfills';
 
+// import default stylesheet with variables
+require('font-awesome/css/font-awesome.css');
+require('mastodon-application-style');
+
+require.context('../images/', true);
+
```

Comment:
This looks like it's not needed here, as the file is never used with multi-frontends? Maybe better we remove this to avoid possible conflicts